### PR TITLE
File management

### PR DESCRIPTION
**Pull request by sk057**

### CODE REVIEW

File: Controllers/FileReader.cs [hunk 1]
```diff
@@ -17,34 +18,38 @@ public void Import()
...
-                        ReadSEB(reader, quotes);
-                        break;
-                    case "Operacijos/B":
-                    case "Transaction ":
-                        ReadLum(reader);
-                        break;
-                    default:
-                        MessageBox.Show("Wrong file type");
-                        break;
+                    switch (line.Substring(0, 12))
```
Comment (AleksandrasMaliuginas) [inline Controllers/FileReader.cs:26]: Exception will be raised if line is shorter then 12

**Review by AleksandrasMaliuginas — COMMENTED** [review]
It's very difficult to comprehend the code. We should discuss how to improve code readability.

File: Controllers/FileReader.cs (hunk 1 above)
Comment (sk057) [reply Controllers/FileReader.cs:26]: This isn't the final commit I only wanted to know if the code worked on your computer

File: Controllers/FileWriter.cs [hunk 2]
```diff
@@ -9,9 +9,10 @@ internal class FileWriter
     {
         public void Export()
         {
-            var saveFileDialog = new SaveFileDialog();
-            var filter = "CSV file (*.csv)|*.csv| All Files (*.*)|*.*";
-            saveFileDialog.Filter = filter;
+            var saveFileDialog = new SaveFileDialog
+            {
+                Filter = @"CSV file (*.csv)|*.csv| All Files (*.*)|*.*"
```
Comment (AleksandrasMaliuginas) [inline Controllers/FileWriter.cs:14]: Maybe it is better to set Filter to `@"CSV file (*.csv)|*.csv"` since our application accepts only csv format?

**Review by AleksandrasMaliuginas — APPROVED** [review]

**Pull request merged**